CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # bedrock-vue-wallet ChangeLog
 
+## 21.2.1 - 2023-10-xx
+
+### Added
+- Add "Settings" and "Log Out" options to header & drawer.
+
+### Changed
+- Move establishing icon library to vue component in order to run code locally with npm link.
```

Comment:
This is a breaking change. Any application that uses this library but doesn't use the `WalletLayout` component will need to set the default icon set now, right? Do we need to do this?

Will it mean that every component needs to import / set the icon library? If not, which ones should do it and how should it be decided? Will this be an easy decision for developers? Is this new approach good ... bad? Pros/cons?

Couldn't the code already be run locally without making this change?

Reply:
I was unable to run the code locally without this change. This is the error from the console that prevents the app from loading. For some reason the _Quasar_ variable is undefined in this function and is unable to set the iconSet. So I looked for alternative ways to set the icons. Seeing that WalletLayout utilizes the q-layout component from Quasar which is ["...designed to manage the entire window and wrap page content..."](https://quasar.dev/layout/layout/), I thought it would be the best solution to establish the icon set. 

**I could very easily not be seeing the whole picture**, but I didn't expect a use case for an application to want to use this library without the WalletLayout component since this is a UI library made to generate a wallet UI?

This is the console error if you run the code prior to the changes in this PR:
![Screenshot 2023-10-06 at 10 38 26 AM](https://github.com/digitalbazaar/bedrock-vue-wallet/assets/56396286/183d0f75-0187-4733-ad48-d36875b4bbb8)

This is the line of code in bedrock-vue-wallet > lib > index.js (line 58 before changes from this PR)
![Screenshot 2023-10-06 at 10 40 05 AM](https://github.com/digitalbazaar/bedrock-vue-wallet/assets/56396286/4e24cb40-436b-49a1-8f31-426d936999ae)

If that line is commented out the app will render, but without several icons:
<img src="https://github.com/digitalbazaar/bedrock-vue-wallet/assets/56396286/3ccab180-db01-495e-b99a-699cf16e0c47" height="400" />

<img src="https://github.com/digitalbazaar/bedrock-vue-wallet/assets/56396286/0c779644-7f4e-4c47-af5e-10f62a7fd0fc" height="400" />

Would value any and all feedback and perspectives here!

Reply:
The breaking change notated at the top of this comment has been included in the CHANGELOG.md b9972b0

Reply:
I reverted the change made to iconSet and updated the change log so there should be no breaking changes with this PR 2e4fd42